src/utils/logging.c

```diff
@@ -37,7 +37,10 @@ gboolean bd_utils_init_logging (BDUtilsLogFunc new_log_func, GError **error __at
     /* XXX: the error attribute will likely be used in the future when this
        function gets more complicated */
 
-    log_func = new_log_func;
+    if (new_log_func == NULL)
+        log_func = &bd_utils_log_stdout;
```

Comment:
I think this should just assign `NULL` enabling a nice way to disable logging altogether.

Reply:
Ok, fixed.